tests/Doctrine/Tests/Mocks/ConnectionMock.php

```diff
@@ -4,9 +4,24 @@
 
 class ConnectionMock extends \Doctrine\DBAL\Connection
 {
+    /**
+     * @var
+     */
     private $_fetchOneResult;
```

Comment:
@morozov What should we use here?

Reply:
Likely mixed, try checking for the setter calls to discover its type.

Reply:
I guess is mixed. I can't find a single reference, even in `Doctrine\DBAL\Connection` :disappointed:

Reply:
@Ocramius We are discussing the missing `mixed` here :smile:

Reply:
If we remove it together with `setFetchOneResult()` and it doesn't break tests, I'd consider it the correct approach.

Reply:
Is up to you guys: `mixed`, or remove `fetchColumn` and `setFetchOneResult` methods?

Reply:
@carusogabriel please remove the unused methods if it doesn't affect existing tests.